charts/vector/ci/initContainers-values.yaml

```diff
@@ -0,0 +1,14 @@
+# Including initContainers
+## Values file for testing adding initContainers to Vector Pods.
+initContainers:
+  - name: sleep
+    image: busybox
+    command: ['sh', '-c', "sleep 5"]
+  - name: wget
```

Comment:
Why is this called wget?

Reply:
😂 it was originally wget'ing something, based on an example. I updated it to not depend on external systems. I'll rename